src/view/element.js

```diff
@@ -53,8 +53,9 @@ export default class Element extends Node {
 	 * {@link module:engine/view/downcastwriter~DowncastWriter#createAttributeElement} for inline element,
 	 * {@link module:engine/view/downcastwriter~DowncastWriter#createContainerElement} for block element,
 	 * {@link module:engine/view/downcastwriter~DowncastWriter#createEditableElement} for editable element,
-	 * {@link module:engine/view/downcastwriter~DowncastWriter#createEmptyElement} for empty element or
-	 * {@link module:engine/view/downcastwriter~DowncastWriter#createUIElement} for UI element instead.
+	 * {@link module:engine/view/downcastwriter~DowncastWriter#createEmptyElement} for empty element,
+	 * {@link module:engine/view/downcastwriter~DowncastWriter#createUIElement} for UI element instead or
+	 * {@link module:engine/view/upcastwriter~UpcastWriter#createElement} for general element creation.
```

Comment:
I'd remove this documentation from here (and move it to the class docs if that information is missing there). Let's keep the maximum in the class definition. This method is protected so not many people will see it. 

Please also review other element/text/df classes for the same issue.